src/reducers/userProjectsByUserNameReducer.js

```diff
@@ -0,0 +1,22 @@
+import {GET_PROJECT_BY_FIRSTNAME_AND_LASTNAME, USER_NOT_FOUND_ERROR} from '../constants/userProfile';
+
+const initialUserProject = {
+  userProjects: [],
+  userError: null,
+};
+
+export const userProjectsByUserNameReducer = (state = initialUserProject, action )=>{
+  switch (action.type) {
+    case GET_PROJECT_BY_FIRSTNAME_AND_LASTNAME : 
+    return {
+      ...state,
+      projects: action.payload
```

Comment:
I would suggest to rename `projects` to `userProjects` in order to keep the naming consistent. Also, I would add `action.payload.projects` since the payload would be coming like this :

`{
  "message": "Found projects",
  "projects": [...]
}
`
these makes it so the `state.projects` would hold just the projects data

Reply:
@j-corp-25 This is a great suggestion, would definitely try to implement it.